docs/UserGuide.md

```diff
@@ -241,13 +144,14 @@ _Details coming soon ..._
 
 ## Command summary
 
-| Action     | Format, Examples                           |
-|------------|--------------------------------------------|
-| **Add**    | `add NAME`<br/> e.g., `add Grilled Salmon` |                                                                                                                                                             |            |                                      |
-| **List**   | `list`                                     |                                                                                                                                                                                                                                                                                                                             |            |                                      |
-| **View**   | `view INDEX`<br/> e.g., `view 2`           |
-| **Delete** | `delete INDEX`<br> e.g., `delete 3`        |                                                                                                                                                                          |            |                                      |
-| **Exit**   | `exit`                                     |   
+| Action     | Format, Examples                                                                                                                                                                                                      |
+|------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Add**    | `add n/RECIPE_NAME d/RECIPE_DURATION p/RECIPE_PORTION [t/RECIPE_TAGS] [s/RECIPE_STEPS] [i/RECIPE_INGREDIENTS]`<br/><br/> e.g., `add n/Chicken Noodles d/20 minutes p/1-2 people i/chicken thigh i/noodle i/soy sauce` |                                                                                                                                                             |            |                                      |
```

Comment:
need to update the command format here also for `add` and `delete` (wrt the changes specified above)